src/index.js

```diff
@@ -1,8 +1,50 @@
 import './index.css';
-import addUser from './modules/AddUser.js';
+import createNewGame from './modules/newGame.js';
+import onloadScore from './modules/onLoadScore.js';
+import refreshScores from './modules/get.js';
 
 const submitBtn = document.getElementById('submitBtn');
+const refreshBtn = document.getElementById('refreshBtn');
+const game = 'game';
+
+const getGameId = async () => {
+  if (localStorage.getItem('gameId')) {
+    return localStorage.getItem('gameId');
+  }
+  return createNewGame(game);
+};
+
+const gameId = await getGameId();
 
 submitBtn.addEventListener('click', () => {
-  addUser();
-});
\ No newline at end of file
+  onloadScore(gameId);
```

Comment:
- Optional
When the app is loaded initially, the score list should be populated on the UI. The function of the refresh button is to only fetch the data when we post a score and update the UI with new scores. I would strongly recommend you implement this feature.